File: Secuencial/MacroBloque.py
class MacroBloque:

    #Crea un macrobloque a partir de un arreglo de pixeles, y la posición de este macrobloque.
    def __init__(self, arr, x, y):
        self.arr = arr
        self.x = x
        self.y = y
    
    #Compara este macrobloque con otro macrobloque
    def comparar(self, MB):
        contadorCoincidencias = 0
        for i in range(len(MB.arr)):
            if(self.arr[i] != MB.arr[i]):
                contadorCoincidencias = contadorCoincidencias+1
        return contadorCoincidencias

File: SecuencialC++/macrobloque.cpp
#pragma once
#include <vector>

using namespace std;

class macrobloque
{
  public:
  int x,y;
  vector<string>arr;
  macrobloque(){}
  macrobloque(vector<string>arr,int x, int y){
    this->arr=arr;
    this->x=x;
    this->y=y;
  }

  int comparar(macrobloque MB){
      int contadorCoincidencias=0;
      for(int i=0; i<this->arr.size();i++){
          if(this->arr!=MB.arr){
              contadorCoincidencias++;
          }
      }    
      return contadorCoincidencias;  
  }

};